gulpfile.js

```diff
@@ -82,6 +83,58 @@ function createReloadServer() {
   });
 }
 
+function writeServiceWorkerFile() {
+  var path = require('path');
+  var rootDir = 'dist';
+
+  var config = {
+    cacheId: 'polymerproject',
+    staticFileGlobs: [
+      rootDir + '/1.0/index.html',
```

Comment:
If you're runtime caching, I think the SW is just caching the contents of the stuff it pulls down over the network, which is no problem.

The problem with static files is that, at build time, sw-precache caclulates a hash from the contents of the static file in your local build, and bakes that into the service-worker.js code it generates. 

So it's actually caching something like:

```
index.html?sw-precache=<some-hash-string>
```

If it finds that file in its cache, it assumes it's up to date if <some-hash-string> matches the hash string it expects.

I think. I actually broke my brain a little bit on this.